lib/version/version.c.in

```diff
@@ -15,8 +15,6 @@ char* __cmake_build_options = " CMAKE_BUILD_TYPE: @CMAKE_BUILD_TYPE@ \n"
                               " USE_OLD_DPDK: @USE_OLD_DPDK@\n"
                               " - RTE_SDK: @RTE_SDK@\n"
                               " - RTE_TARGET: @RTE_TARGET@\n"
-                              " USE_HSA: @USE_HSA@\n"
-                              " USE_CLOC: @USE_CLOC@\n"
```

Comment:
Maybe add in the ASDF and GDAL flags since we also did that in the readme, and we are thinking about it.